docs/components/ProgressBar/Mobile.stories.tsx

```diff
@@ -16,9 +16,38 @@ const BasicTemplate: ComponentStory<typeof ProgressBar> = args => (
   <ProgressBar {...args} />
 );
 
+const WithHeaderTemplate: ComponentStory<typeof ProgressBar> = args => {
+  const [step, setStep] = useState(1);
+  const totalSteps = 6;
+  return (
+    <ProgressBar
+      {...args}
+      current={step}
+      total={totalSteps}
+      header={
+        <>
+          <Heading level="heading">Upload progress</Heading>
+          <br />
```

Comment:
Overall looks good. Since `br` isn't in react-native it may be worth using content to wrap the Header handle this
https://job-73344-job-74550-update-p.atlantis.pages.dev/?path=/docs/components-layouts-and-structure-content-docs--page